USER_GUIDE.md

```diff
@@ -60,6 +60,10 @@ redirect: https://raw.githubusercontent.com/openfaas/faas/master/.DEREK.yml
 
 If `dco_check` is specified in the feature list then Derek will inform you when a PR is submitted with commits which have no sign-off. He also adds a label `no-dco`.
 
+### Feature: `no_empty_pr`
+
+If `no_empty_pr` is specified in the feature list then Derek will inform you that a PR needs a description. He also adds the `invalid` label.
```

Comment:
I'm not sure if `validate_pr_description` / `validate_pr_text` would be more descriptive?

Reply:
Not a problem, the feature name wasn't defined in the issue, so I did my best to succinctly describe the use as the feature name. I was trying to keep the feature name small to reduce the potential for typing errors

Reply:
`pr_content_check`

Reply:
updated. Concise and clear. Thanks @rgee0!